### PR TITLE
Use chunking with compression in regular stress tests.

### PR DESCRIPTION
**Pull request by andre4i**
## Description

This change enables the stress tests to send a large op every N regular ops. We will start with N == 500. Every 500 small ops, the test will send a payload of up to 2172864 bytes which will then get compressed and chunked. In total, this change will add 20 large ops for each of the 100 clients. That would be 2000 large ops in total for each stage.

Tested via the private branch https://github.com/microsoft/FluidFramework/compare/test/andreiiacob. Results here: https://dev.azure.com/fluidframework/internal/_build/results?buildId=127829&view=results

### CODE REVIEW

**Comment by vladsud** [comment]
> will then get compressed and chunked

how do we validate that? I.e. we know that something happened (test did not fail). But how do we know that (for example) chunking is being tested? I.e. maybe compression is all that is sufficient to get under the limit and we have no coverage for chunking?

Not sure how to answer these, but let's ensure that we use much smaller limits (not 1Mb) as a forcing function (plus some other validation that indeed both features are being tested).

File: packages/test/test-service-load/src/loadTestDataStore.ts [hunk 1]
```diff
@@ -573,27 +573,20 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
...
-				  }
-				: () => {
-						const opPayload = generateContentOfSize(getOpSizeInBytes());
-						const opKey = Math.random().toString();
-						dataModel.sharedmap.set(opKey, opPayload);
-				  };
+		const sendSingleOp = () => {
+			if (opSizeinBytes > 0 && largeOpRate > 0 && opsSent % largeOpRate === 1) {
+				const opPayload = generateContentOfSize(getOpSizeInBytes());
+				const opKey = Math.random().toString();
```
Comment (vladsud) [inline packages/test/test-service-load/src/loadTestDataStore.ts:582]: use uuid() instead?

File: packages/test/test-service-load/src/loadTestDataStore.ts [hunk 2]
```diff
@@ -573,27 +573,20 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
...
+				dataModel.sharedmap.set(opKey, opPayload);
+			} else {
+				dataModel.counter.increment(1);
```
Comment (vladsud) [inline packages/test/test-service-load/src/loadTestDataStore.ts:585]: I think you want to do both. I.e. if you take first branch always, will test complete? Maybe it does today (I remember completion logic was weird), but I think we should expect some number of increments on the counter as a main thing here, and thus we should keep counting...

**Review by vladsud — APPROVED** [review]

**Comment by andre4i** [comment]
> how do we validate that? I.e. we know that something happened (test did not fail). But how do we know that (for example) chunking is being tested? I.e. maybe compression is all that is sufficient to get under the limit and we have no coverage for chunking?

The op content is a random string which won't compress at all. So, if the payload exceeds the limit, it will definitely do chunking. The same ideas as in https://github.com/microsoft/FluidFramework/blob/65a7a49c82caa09e8af9a6473a5802096f861e66/packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

File: packages/test/test-service-load/src/loadTestDataStore.ts (hunk 2 above)
Comment (andre4i) [reply packages/test/test-service-load/src/loadTestDataStore.ts:585]: The test completes, see https://dev.azure.com/fluidframework/internal/_build/results?buildId=127829&view=results

I believe the completion logic relies on the opsSent variable: https://github.com/microsoft/FluidFramework/blob/main/packages/test/test-service-load/src/loadTestDataStore.ts#L629

This needs to be decoupled, as if we only rely on the counter value (setting aside the problem of using the subject of the test for coordination), the large ops will not be accounted for, so the test would technically send 20 extra ops per each client.

File: packages/test/test-service-load/src/loadTestDataStore.ts [hunk 3]
```diff
@@ -573,27 +574,18 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
...
-						dataModel.counter.increment(1);
-				  }
-				: () => {
-						const opPayload = generateContentOfSize(getOpSizeInBytes());
-						const opKey = Math.random().toString();
-						dataModel.sharedmap.set(opKey, opPayload);
-				  };
+		const sendSingleOp = () => {
+			if (opSizeinBytes > 0 && largeOpRate > 0 && opsSent % largeOpRate === 1) {
+				dataModel.sharedmap.set(uuid(), generateContentOfSize(getOpSizeInBytes()));
```
Comment (anthony-murphy) [inline packages/test/test-service-load/src/loadTestDataStore.ts:582]: do we actually want a random key? this will cause each large op to increase to total payload size for snapshots, which seems orthogonal to this change. ideally we could run stress forever, and not have it fail, but this grows unbound so will eventually get too large.

File: packages/test/test-service-load/testConfig.json [hunk 4]
```diff
@@ -36,7 +36,11 @@
 						"Fluid.Container.summarizeProtocolTree2": [true, false]
 					}
 				}
-			}
+			},
+			"useRandomContent": true,
```
Comment (anthony-murphy) [inline packages/test/test-service-load/testConfig.json:40]: should these be grouped, its not clear that they are related when flatly listed on the config
